### ISSUE TITLE
"Archive" levels from view on Memberships > Membership Levels in dashboard.

### ISSUE THREAD

**Issue by kimcoleman** [strangerstudios/paid-memberships-pro#785]
A member has requested a way to hide some now unused levels from view in their admin area. The area is overwhelmed with levels and she'd like to clean it up.

We could consider a way to either use custom code or an "archive" setting for levels that are no longer used. Alternately, we could offer a toggle to show/hide members that are set to "hidden" (not allow signups" from view.

**Comment by ideadude** [comment]
I don't think we would build this in core, but I could see a gist for it and we can add css classes and filters if needed to support that.

It could use CSS to hide levels marked as "No" for allow signups. And then show a link to show those levels. Could do almost entirely in JS if we added a css class to the rows.

**Comment by ideadude** [comment]
Kind of hacky, but we already add a CSS class to rows with disabled signups: pmpro_gray

So this could be done as a gist that inserts JS into the page that hides the rows and inserts the link to show them.

**Comment by dparker1005** [comment]
> Alternately, we could offer a toggle to show/hide members that are set to "hidden" (not allow signups" from view.

I think that  is a cool approach. Only show checkbox if there are hidden levels to begin with
